src/inet/UDPEndPointImplOpenThread.cpp

```diff
@@ -28,7 +28,9 @@
 #include <system/SystemPacketBuffer.h>
 
 // Temporary until we manage to update the OT version in the CI
-#ifndef OT_NETIF_THREAD_HOST
+// Define a macro if OT_NETIF_THREAD_HOST is present in the enum
+#define HAS_OT_NETIF_THREAD_HOST (OT_NETIF_THREAD_HOST == OT_NETIF_THREAD_HOST)
+#ifndef HAS_OT_NETIF_THREAD_HOST
```

Comment:
the `#ifdef` ins't correct anymore. `HAS_OT_NETIF_THREAD_HOST` will always defined.
```suggestion
#if !HAS_OT_NETIF_THREAD_HOST
```

Reply:
How was it able to build from CSA then? Are the new ot Libs already there?